assets/variant-list.css

```diff
@@ -45,10 +45,10 @@ variant-list .quantity__button {
     overflow-y: auto;
   }
 
```

Comment:
Products with 1 variant, no variants that dont have the bottom bar are getting this error: https://screenshot.click/04-46-9dpo9-ad5e8.png

Reply:
Updated - https://github.com/Shopify/dawn/pull/2754/commits/5dd7a331416bb68a8d5eb0f07e8f8888609f4aef